addons/website_membership/controllers/main.py

```diff
@@ -30,14 +28,13 @@ class WebsiteMembership(http.Controller):
         '/members/association/<membership_id>/country/<int:country_id>/page/<int:page>',
     ], type='http', auth="public", website=True)
     def members(self, membership_id=None, country_name=None, country_id=0, page=1, **post):
-        cr, uid, context = request.cr, request.uid, request.context
-        product_obj = request.registry['product.product']
-        country_obj = request.registry['res.country']
-        membership_line_obj = request.registry['membership.membership_line']
-        partner_obj = request.registry['res.partner']
+        Product = request.env['product.product']
+        MembershipLineSudo = request.env['membership.membership_line'].sudo()
+        membership_lines = request.env['membership.membership_line'].sudo()
```

Comment:
good idea @jaa-odoo  use in one line : 
MembershipLineSudo = membership_lines = request.env['membership.membership_line'].sudo()